### PR TITLE
History Pruning Tweak

### PR DESCRIPTION
**Pull request by VoyagerOne**
Don't prune main killer move.
Increased pruned depth to 4.

STC:
LLR: 2.96 (-2.94,2.94) [0.00,5.00]
Total: 23380 W: 4581 L: 4350 D: 14449

LTC:
LLR: 2.96 (-2.94,2.94) [0.00,5.00]
Total: 28934 W: 4329 L: 4105 D: 20500

Bench:  8642100

### CODE REVIEW

**Comment by Rocky640** [comment]
Well played, GM VO !

**Comment by VoyagerOne** [comment]
HOO-RAH!!!

Thanks snicolet & Rocky.
More ELO to discover with History Pruning...

**Comment by joergoster** [comment]
Congrats!

**Comment by zamar** [comment]
approved

**Comment by ajithcj** [comment]
Congrats VoyagerOne!

**Comment by locutus2** [comment]
Congrats too!

File: src/search.cpp [hunk 1]
```diff
@@ -923,7 +923,8 @@ namespace {
               continue;
 
           // History based pruning
-          if (   depth <= 3 * ONE_PLY
+          if (   depth <= 4 * ONE_PLY
+              &&  move != ss->killers[0]
```
Comment (hxim) [inline src/search.cpp]: formatting: i think there should be only one space before "move" aligned left with depth on previous line

**Comment by VoyagerOne** [comment]
Ok, I removed the extra space.

**Pull request closed without merging**